ocs_ci/ocs/ui/validation_ui.py

```diff
@@ -433,16 +451,53 @@ def odf_storagesystems_ui(self):
                 enable_screenshot=True,
             )
         logger.info("Click on Overview tab")
-        self.do_click(self.validation_loc["overview"], enable_screenshot=True)
+        if (
+            self.ocp_version_semantic == version.VERSION_4_11
+            and self.ocs_version_semantic == version.VERSION_4_10
+        ):
+            self.do_click(
+                self.validation_loc["overview_odf_4_10"], enable_screenshot=True
+            )
+        else:
+            self.do_click(self.validation_loc["overview"], enable_screenshot=True)
         logger.info("Click on 'Object' tab")
-        self.do_click(self.validation_loc["object"], enable_screenshot=True)
+        if (
+            self.ocp_version_semantic == version.VERSION_4_11
+            and self.ocs_version_semantic == version.VERSION_4_10
+        ):
+            self.do_click(
+                self.validation_loc["object-odf-4-10"], enable_screenshot=True
+            )
+        else:
+            self.do_click(self.validation_loc["object"], enable_screenshot=True)
         if not config.ENV_DATA["mcg_only_deployment"]:
             logger.info("Click on 'Block and File' tab")
-            self.do_click(self.validation_loc["blockandfile"], enable_screenshot=True)
+            if (
+                self.ocp_version_semantic == version.VERSION_4_11
+                and self.ocs_version_semantic == version.VERSION_4_10
+            ):
+                self.do_click(
+                    self.validation_loc["blockandfile-odf-4-10"], enable_screenshot=True
+                )
+            else:
+                self.do_click(
+                    self.validation_loc["blockandfile"], enable_screenshot=True
+                )
         if not config.DEPLOYMENT.get("external_mode"):
             if not config.ENV_DATA["mcg_only_deployment"]:
                 logger.info("Click on 'BlockPools' tab")
-                self.do_click(self.validation_loc["blockpools"], enable_screenshot=True)
+                if (
```

Comment:
What happens with 4.12?

Reply:
For 4.12 or any other version, it will execute the line in the else part.